ClassTranscribeServer/Controllers/LogsController.cs

```diff
@@ -316,10 +384,12 @@ public class StudentLog
         public class MediaLog
         {
             public string MediaId { get; set; }
+            public string MediaName { get; set; }
```

Comment:
Add Duration too?

Reply:
@ruihuasui do you think we should add duration? Would it be helpful to have?

Reply:
Yeah, It sounds good to have a duration.